code/modules/projectiles/guns/launcher/money_cannon.dm

```diff
@@ -0,0 +1,166 @@
+/obj/item/weapon/gun/launcher/money
+	name = "money cannon"
+	desc = "A blocky, plastic novelty launcher that claims to be able to shoot thaler at considerable velocities."
+	icon_state = "money_launcher"
+	item_state = "money_launcher"
+	origin_tech = list(TECH_COMBAT = 1, TECH_MATERIAL = 1)
+	slot_flags = SLOT_BELT
+	w_class = ITEM_SIZE_SMALL
+	release_force = 80
+	fire_sound_text = "a whoosh and a crisp, papery rustle"
+	fire_delay = 1
+	fire_sound = 'sound/weapons/guns/fire/money_launcher.ogg'
+	var/emagged = 0
+
+	var/receptacle_value = 0
+	var/dispensing = 20
+
+/obj/item/weapon/gun/launcher/money/hacked
+	emagged = 1
+
+/obj/item/weapon/gun/launcher/money/proc/vomit_cash(var/mob/vomit_onto, var/projectile_vomit)
```

Comment:
nani the fuck

Reply:
Proc for "suiciding" with money cannon. Causes you basically shoot money in your mouth, all around and play casino win sound.